packages/esm-patient-programs-app/src/programs/program-actions-menu.component.test.tsx

```diff
@@ -0,0 +1,70 @@
+import { showModal } from '@openmrs/esm-framework';
+import { ProgramsActionsMenu } from './programs-actions-menu.component';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import React from 'react';
+import { launchPatientWorkspace } from '@openmrs/esm-patient-common-lib';
+
+jest.mock('@openmrs/esm-framework', () => ({
+  showModal: jest.fn(),
+  useLayoutType: jest.fn(() => 'desktop'),
+}));
+
+jest.mock('@openmrs/esm-patient-common-lib', () => ({
+  launchPatientWorkspace: jest.fn(),
+}));
```

Comment:
Same here for this test case about mocking patterns

Reply:
This is fine for `@openmrs/esm-patient-common-lib`, @NethmiRodrigo.